sdk/attestation/azure-security-attestation/azure/security/attestation/_administration_client.py

```diff
@@ -88,6 +103,7 @@ def set_policy(self, attestation_type, attestation_policy, signing_key=None, **k
         :param Optional[AttestationSigningKey] signing_key: Optional signing key to be
```

Comment:
signing_key should be a keyword. If you are setting a default. If the default is None you don't need to include it in the method signature and can get the value from kwargs.